app/frontends/ogc-coverages/get-coverage-rangeset.ts

```diff
@@ -90,6 +90,9 @@ export default function getCoverageRangeset(
       varInfo.variables, varInfo.coordinateVariables);
   }
 
+  if (query.extend) {
+    validateExtend(query.extend, req.params.collectionId, req.collections, varInfos);
```

Comment:
Since we're only updating the OGC coverages API that means the parameter will not be supported from our WMS AMI - I think that's fine, just making note of it.